### PR TITLE
kata-webhook: request cpu/memory to the pod and a few improvements

### PR DESCRIPTION
**Pull request by wainersm**
Primarily this should fix a test case which is failing on OpenShift CI (see #3858). While I am at it, I improved the `create-certs.sh` script a little bit.

### CODE REVIEW

File: kata-webhook/deploy/webhook.yaml [hunk 1]
```diff
@@ -33,6 +33,10 @@ spec:
             - name: webhook-certs
               mountPath: /etc/webhook/certs
               readOnly: true
+          resources:
```
Comment (egernst) [inline kata-webhook/deploy/webhook.yaml:36]: seems fine -- any particular reason for 1 CPU instead of 100m, or 100 CPU? 

Same for memory

**Review by snir911 — APPROVED** [review]
LGTM, thanks, also fine with lowering the requests

**Review by GabyCT — APPROVED** [review]

**Comment by GabyCT** [comment]
@wainersm please put the label if this PR needs or not a backport thanks

File: kata-webhook/deploy/webhook.yaml (hunk 1 above)
Comment (wainersm) [reply kata-webhook/deploy/webhook.yaml:36]: @egernst for no particular reason. But maybe it should be better set 100m?

Comment (wainersm) [reply kata-webhook/deploy/webhook.yaml:36]: Just updated this PR; setting "cpu==100m" and "memory==205Mi", which should be enough.

**Comment by GabyCT** [comment]
@wainersm does this needs backport?

**Pull request merged**